### ISSUE TITLE
Can CHANGELOG.md  be mapped to README.md

### ISSUE THREAD

**Issue by soeasyjx** [changesets/changesets#1023]
## mapped to README.md

将包发布到npm上，有关包的信息是读取README.md文件，但是changeset version命令是生成名为：CHANGELOG.md ，我不知道现在是否有方法可以让生成名为READMD.md

**Comment by airtonix** [comment]
I would instead just do this after `changset version` : 


```sh
changeset version
CHANGELOG=$(cat ./CHANGELOG.md) envsubst < ./README.md.tmpl > ./README.md
git add ./README.md
git commit -am "Merge CHANGELOG into README"
git push origin HEAD
```

**Comment by airtonix** [comment]
If you're looking for a more integrated solution, then you'll have to wait for the outcome of some design decisions.

I believe there was talk of making the behaviour of changesets more composable like babel and eslint are. So in theory you might be able to swap out the part that writes the file with your own.

**Comment by Andarist** [comment]
Yeah, we try to make it more flexible - but sometimes too flexible is also a problem and we want to avoid this. We don't plan to allow what you are asking for here. However, as @airtonix has suggested - it's possible to script this stuff on top of Changesets. Don't be afraid of adding some pre/post scripts
